bin/create_release.sh

```diff
@@ -44,8 +44,16 @@ if [[ ! $bamboo_RELEASE_FLAG == true ]]; then
   exit 0
 fi
 
+## Check if the code has already been released.
+export url="https://github.com/nasa/cumulus-orca/releases/tag/"v$bamboo_ORCA_VERSION""
+if curl --output /null --silent --fail "$url"; then
+  echo "Release URL already exists: $url. Exiting."
+  exit 0
```

Comment:
Good check. I do lean towards having this be an `exit 1` to make the failure a bit noisier.

Reply:
ok fair enough.